docker-compose.yml

```diff
@@ -56,6 +56,24 @@ services:
     networks:
       - bcros
 
+####################### Lear DB Test Fixture Service Definition #######################
```

Comment:
This file needs to be moved to somewhere in a dev support structure, as it;'s not how the app is deployed and is confusing at this level. I know this isn't part of this PR, but is part of the body of work that was done.

Reply:
I'm just creating this new service as a containerized service for local development and as a tool that can be built locally by QA resources. There is a pretty extensive and clear Readme file to prevent confusion.